libs/application/templates/children-residence-change/src/forms/ParentBForm.ts

```diff
@@ -55,6 +56,17 @@ export const ParentBForm: Form = buildForm({
         }),
       ],
     }),
+    buildSection({
+      id: 'interviewParentB',
```

Comment:
Should we use the imported type here?

Reply:
This id doesn't do anything so I left it out on purpose..